options.go

```diff
@@ -127,6 +127,12 @@ type Options struct {
 	maxBatchSize  int64 // max batch size in bytes
 
 	maxValueThreshold float64
+
+	testChan chan string
```

Comment:
what is this used for compared to onCloseDiscardCapture? Can we not just live with one var?

Reply:
testChan is used currently for two messages: 
1. The end of vlog.init(db)  (see line 558 of value.go)
2. The end of one cycle of updateDiscardStats (value.go: 1117)

We use this channel here to synchronize between go-routines instead of using time.Sleep (and simply praying for synchronization).

Note that the onCloseDiscardCapture map is *not* something we wait for! Instead, it is simply populated automatically at the time that we invoke valueLog.Close(db). (Called from DB.Close()). Since DB.Close() is synchronous, we don't have to wait for this data via any sort of special channel.

Can we live with just one of these? Theoretically, we could populate a string with the map contents and push it to the testChan. However, I do not recommend this approach. The two additions: the testChan and onCloseDiscardCapture have distinct requirements.